bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/internal/VoiceManagerImpl.java

```diff
@@ -941,8 +941,8 @@ public Collection<KSService> getKSs() {
     }
 
     @Override
-    public List<HumanLanguageInterpreter> getHLIsByIds(String ids) {
-        return getHLIsByIds(Arrays.asList(ids.split(",")));
+    public List<HumanLanguageInterpreter> getHLIsByIds(@Nullable String ids) {
+        return ids == null ? new ArrayList<>() : getHLIsByIds(Arrays.asList(ids.split(",")));
```

Comment:
This code is shorter and returns a constant which prevents unneccessary garbage collection.

```suggestion
        return ids == null ? List.of() : getHLIsByIds(Arrays.asList(ids.split(",")));
```

Reply:
I didn't know about that, makes sense. Thanks for the information!